store_monitoring/crud.py

```diff
@@ -41,4 +51,40 @@ def insert_bq_results(db):
             db.commit()
             db.refresh(data)
     return
-        
\ No newline at end of file
+
+def current_time():
+    current_time=datetime.now()
+    one_hour_ago=current_time - timedelta(minutes=60)
+    time_range={"current_time": current_time, "one_hour_ago": one_hour_ago}
+    return time_range
+
+def get_business_hour(db):
+    data= db.query(model.MenuHours).filter(and_(model.MenuHours.store_id==579100056021594000),(model.MenuHours.day==0)).all()
+    return data
+
+def intersection(intervals):
+    start, end = intervals.pop()
+    while intervals:
+        start_temp, end_temp = intervals.pop()
+        start = max(start, start_temp)
+        end = min(end, end_temp)
+    return [start, end]
+
+def query(db):
+    business_hour=get_business_hour(db)
+    time=current_time()
+    for i in business_hour:
+        start_time=string_to_time(i.start_time_local)
+        end_time=string_to_time(i.end_time_local)
+        intervals=[[time["one_hour_ago"].time(), time["current_time"].time()], [start_time, end_time]]
+        if((start_time<time["one_hour_ago"].time()) and (end_time<time["one_hour_ago"].time())):
```

Comment:
This check can be reduced to one condition.

If end time < one hour ago time, because the other will obviously be true then